### PR TITLE
feat: Update project from project node in explorer

### PR DESCRIPTION
**Pull request by jdneo**

### CODE REVIEW

File: src/views/dependencyDataProvider.ts [hunk 1]
```diff
@@ -55,6 +55,17 @@ export class DependencyDataProvider implements TreeDataProvider<ExplorerNode> {
             commands.executeCommand(Commands.JAVA_BUILD_WORKSPACE)));
         context.subscriptions.push(instrumentOperationAsVsCodeCommand(Commands.JAVA_PROJECT_CLEAN_WORKSPACE, () =>
             commands.executeCommand(Commands.JAVA_CLEAN_WORKSPACE)));
+        context.subscriptions.push(instrumentOperationAsVsCodeCommand(Commands.JAVA_PROJECT_UPDATE, async (node: INodeData) => {
+            if (!node.uri) {
+                return;
```
Comment (Eskibear) [inline src/views/dependencyDataProvider.ts:60]: 1. In what case, node.uri is undefined?
2. In such case, when user click "Update", it simply returns silently? Is it a good experience?

Comment (jdneo) [reply src/views/dependencyDataProvider.ts:60]: 1. Based on the current code logic, this won't happen. 
2. Any suggestions?

Comment (Eskibear) [reply src/views/dependencyDataProvider.ts:60]: 1. ok, then it's fine. 
2. I would suggest to throw an error/warning here, then users would know the action didn't take effect.  (if for some reason node.uri is wrongly set to undefined)

Comment (jdneo) [reply src/views/dependencyDataProvider.ts:60]: I see. Will show a error message instead

**Review by Eskibear — APPROVED** [review]

**Pull request merged**